index.html

```diff
@@ -250,5 +250,25 @@ <h2> About
                 </uli>  
             </div>     
         </article >  
+
+        <article class="contact">
+           
+           <h2 class="head-contact"> Contact me </h2>
+           <p class="contact_p"> 
+            If you have an application you are
+             interested in developing, a feature that
+              you need built or a project that needs 
+              coding. I’d love to help with it 
+           </p>
+
+           <form action="https://formspree.io/f/{form_id}" method="post" >
```

Comment:
- [ ] Please make sure that your form is submitting the input. kindly replace the `form_id` with an id which you generate by going to this site. [formpress](https://formspree.io/html/)
check this 👇
![form](https://user-images.githubusercontent.com/66992901/194126295-c2715588-a706-4dbb-9d20-6a29782856b6.PNG)